examples/cosmos/lightclient/typedefs.qnt

```diff
@@ -13,8 +13,8 @@ module typedefs {
         Commits: Set[str]
     }
 
-    type Blockchain = int -> blockHeader
-    type LightBlockMap = int -> lightBlock
+    type Blockchain = int -> BlockHeader
+    type LightBlockMap = int -> LightBlock
```

Comment:
It address one aspect of that issue. But unless the issue is closed, it also raises issues like being able to immediately differentiate module identifiers, or state variables.

That issue may be overly broad at this point!

Reply:
Also, I just realized that it won't work for type variables in annotations (only in type definitions). So yeah, even on that aspect, not a complete solution.

Reply:
True. Thinking about this a bit more, I'm also not sure how much this addresses that problem space. We already have lexical differentiation between types and types variables (upper vs. lower case). I guess if this were less subtle, using something like they do in OCaml, then maybe it would be less easy to make this mistake? I'm not sure.

Reply:
I remember seeing that somewhere, and I just checked: the manual mentions type variables as `a`, ..., `z`

Reply:
I think remove the ability to have descriptive type variables is not great. You could require explicit quantification everywhere?

Reply:
hmm, good point.